_episodes/02-licensing.md

```diff
@@ -70,40 +70,40 @@ The 3 main rules of choosing a license:
 
-Permissive licenses do not require that. People can more or less do whatever they want with the code, as long they attribute it to you and leave the license in that code and do not complain when it breaks something.
+Permissive licenses do not require that. People can more or less do whatever they want with the code, as long they attribute it to you, do not remove any license statements that come with the code, and do not complain when it breaks something.
 
-Copyleft licenses require people to release their source code under the same conditions to the world if they want to share it at all. Permissive licenses do not. Copyleft licenses prevent people form just taking the code modifying it and then making a profit. That also includes yourself, if you are not the sole copyright holder. 
+Copyleft licenses require people to use the same license if they share their derived source code with anyone else. Permissive licenses do not. Copyleft licenses prevent people from taking the code, modifying it, and then sharing the modified version with others under a license that forbids sharing or making changes. That also includes yourself, if you are not the sole copyright holder.
 
-Companies do like permissve licenses more, so if you want commerical contributions to your code a permissve license might be advantageous. On the other hand the linux kernel uses a copyleft license and has many commercial contributors. 
+Companies do like permissive licenses more, so if you want commercial contributions to your code, a permissive license might be advantageous. On the other hand, the linux kernel uses a copyleft license and has many commercial contributors.
 
 License compatibility can also impact your choice of license.
 
 ## License compatibility
 
 Today most software depends on external libraries, modules or packages, which have their own licenses. How do these licenses influence or limit your choice of the license?
-This is the realm of license compatibility and it is complictated, because it depends on 
+This is the realm of license compatibility and it is complictated, because it depends on
 - how you package your software
 - how integrated the library is in your code
 - what the specific license of that library requires
 - what the dependencies of that library require
 
 Among permissive licenses you will generally not have many problems, but copyleft licenses make things more difficult
-A nice overview is found [in the Turing Way Boook](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html).
+A nice overview is found [in the Turing Way Book](https://the-turing-way.netlify.app/reproducible-research/licensing/licensing-software.html).
 
 [FOSSA](https://fossa.com/?ref=tldrlegal) is a tool which for some programming languages helps you to scan and ascertain that all requirements are met.
 
 Also, think about the data you might ship together with your project because data has copyright too. So check your data.
```

Comment:
This sentence is vague. What exactly is its purpose? Is there a better way to achieve it?